src/Data/Textile/Formula.elm

```diff
@@ -175,9 +179,8 @@ printingImpacts :
 printingImpacts impacts { printingProcess, heatProcess, elecProcess, surfaceMass, ratio } baseMass =
     let
         surface =
-            -- area (m2) = mass (g) / surfaceMass (g/m2)
-            Mass.inGrams baseMass
-                / Unit.surfaceMassToFloat surfaceMass
+            Unit.surfaceMassToSurface surfaceMass baseMass
+                |> Area.inSquareMeters
                 -- Apply ratio
                 |> (\s -> Split.apply s ratio)
```

Comment:
I don't understand: the current implementation of `Split.apply` is apply : `Float -> Split -> Float`, so the last argument **IS** a `Split`.
Are you saying we should have it reversed so the **FIRST** argument is a `Split` instead?

Reply:
Ok, I think I was confused with the arg names; `s` for surface looks a lot it could be for `split`; maybe rename `s` to `surface` so we read `Split.apply surface ratio`?

Reply:
Good idea 👍